### PR TITLE
7.2 User | Low Level Search Options | LRDOCS-7124

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-7124, no widgets left behind. I might have got this all wrong, but I wanted to take a few minutes and push something along so we don't let this lapse.

### CODE REVIEW

**Comment by lipusz** [comment]
**Started reviewing...** :mag::nerd_face:

File: user/articles/120-search/10-low-level-search-options.markdown [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+**Fields to Return:**
+: Enter the names of the stored fields to be returned from the search engine, in
+a comma-separated list. Leave it blank to return all stored fields are returned.
+
+**Contributors to Include:**
+: Enter the ids of registered search contributors to be included at this search,
+in a comma-separated list. If not set, all registered search contributors are
+applied.
+
```
Comment (lipusz) [inline user/articles/120-search/10-low-level-search-options.markdown]: We can solely mention for now that "Contributors" are components implementing the `com.liferay.portal.search.spi.searcher.SearchRequestContributor` interface (provided by the "com.liferay.portal.search.spi" artifact) which is an extension point (SPI) to intercept search requests and add query parts. Will be documented in the Search developer reference docs later.

Comment (lipusz) [reply user/articles/120-search/10-low-level-search-options.markdown]: See https://issues.liferay.com/browse/LPS-96199?focusedCommentId=1905708&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1905708

Comment (lipusz) [reply user/articles/120-search/10-low-level-search-options.markdown]: In addition, also add that the list is a "comma delimited list of FQCN of SearchRequestContributors.

File: user/articles/120-search/10-low-level-search-options.markdown [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Low Level Search Options
+
+Low level search is a new concept in @product@ version 7.2: it's a search that
+doesn't go through the [Search and Indexing
+Framework](/docs/7-2/frameworks/-/knowledge_base/f/model-entity-indexing-framework),
+which is infrastructure used for searching documents in the Liferay Index.
+
```
Comment (lipusz) [inline user/articles/120-search/10-low-level-search-options.markdown]: @arboliveira Hey André,

Can you please check this part below :point_down: ?

Comment (rbohl) [reply user/articles/120-search/10-low-level-search-options.markdown]: bump @arboliveira

Comment (rbohl) [reply user/articles/120-search/10-low-level-search-options.markdown]: @lipusz we got an answer over in #t-search on slack. Let me know if you're still reviewing this or if it should come back into my queue for updates.
:hourglass_flowing_sand:

Comment (lipusz) [reply user/articles/120-search/10-low-level-search-options.markdown]: Yes, you can update it with André's comments.

**Pull request closed without merging**